core/templates/dev/head/pages/error/error.html

```diff
@@ -67,10 +67,8 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
-    @media (max-width: 768px) {
```

Comment:
@llKCll Any rationale behind deleting this code? I am afraid it breaks the mobile view. 

![screenshot from 2017-05-23 17-19-28](https://cloud.githubusercontent.com/assets/24438869/26353051/029499fc-3fdc-11e7-8cb6-4ac7188bff92.png)